ibis/backends/base/sql/__init__.py

```diff
@@ -90,9 +92,8 @@ def table(self, name: str, database: str | None = None) -> ir.Table:
             )
         qualified_name = self._fully_qualified_name(name, database)
         schema = self.get_schema(qualified_name)
-        node = ops.DatabaseTable(
-            name, schema, self, namespace=ops.Namespace(database=database)
-        )
+        namespace = ops.Namespace(database=database)
+        node = ops.DatabaseTable(name, schema, self, namespace=namespace)
```

Comment:
This change seems unrelated to splitting. I'd like to keep these as surgical as possible and not do any "cleaning while in the area" changes until we square the whole thing away.

Can you revert this bit?

Reply:
Sure, picked it accidentally.